### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [airmeat13/prj-rev-bwfs-dasmoto#5]
Don't forget to include the alt attribute it is used both for [accessibility](https://www.codecademy.com/articles/accessibility) and as a backup in case your image cannot be rendered.

In this case, your line of code would look something like:
`    <img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-2/hacksaw.jpeg" alt="Hacksaw">
`

https://github.com/airmeat13/prj-rev-bwfs-dasmoto/blob/ab2d79362e93121afcd84953a3b79726a506bc0f/arts/index.html#L11
